examples/4337-gas-metering/README.md

```diff
@@ -10,14 +10,14 @@ NOTE: If you run a paymaster analysis twice or more without changing the salt fo
 
 ## Gas Usage Results
 
-|                                                                  | **With 4337?** |                                                                                                                 **Account Creation**                                                                                                                  |                                                                                                        **Account Creation + Native Transfer**                                                                                                         |                                                                                                                 **Native Transfer**                                                                                                                  |                                                                                                         **Account Creation + ERC20 Transfer**                                                                                                         |                                                                                                                  **ERC20 Transfer**                                                                                                                  |                                                                                                         **Account Creation + ERC721 Minting**                                                                                                         |                                                                                                                  **ERC721 Minting**                                                                                                                   |
```

Comment:
Should we have a column specifying the Entrypoint version instead of in the name?

Reply:
I think the name is better because it is reusable for detailed runs and provides information about the entrypoint used. If we add a column and remove it from the name, then we'll have an open question about where to add it in the detailed breakdown